automation_oca/models/automation_configuration.py

```diff
@@ -24,9 +24,25 @@ class AutomationConfiguration(models.Model):
     tag_ids = fields.Many2many("automation.tag")
     company_id = fields.Many2one("res.company")
     domain = fields.Char(
-        required=True, default="[]", help="Filter to apply", compute="_compute_domain"
+        required=True,
+        default="[]",
+        help="Filter to apply",
+        compute="_compute_domain",
+        help="""
+        Following special variable can be used in filter :
+         * datetime
+         * dateutil
+         * time """,
```

Comment:
What about `user` and `ref`?

Reply:
you're right @pedrobaeza 
we missed this part, could you have a look @gaelTorrecillas  ?